kubernetes.yml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  generation: 1
+  labels:
+    app: apr
+  name: apr
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    app: apr
+  template:
+    metadata:
+      labels:
+        app: apr
+      name: apr
+    spec:
+      containers:
+      - image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/apr:dbfffa33a052b65a6821ae801d5ecc69064e0e80
+        imagePullPolicy: Always
+        name: apr
+        env:
+        - name: BASIC_AUTH_USER
+          valueFrom:
+            secretKeyRef:
+              name: apr-production-credentials
+              key: username
+        - name: BASIC_AUTH_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: apr-production-credentials
+              key: password
+        - name: PORT
+          value: "4000"
+        - name: KAFKA_CONSUMER_GROUP
+          value: apr-production
+        - name: MIX_ENV
+          value: prod
+        resources: {}
+        securityContext:
+          privileged: false
+        terminationMessagePath: /dev/termination-log
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: apr
+  name: apr
+  namespace: default
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 4000
+  selector:
+    app: apr
+  sessionAffinity: None
+  type: LoadBalancer
```

Comment:
This is so cool if it's truly all that's required to deploy a hosted, horizontally-scaled, auto-recovering service.